recipes/conversation_view/feed_pill/feed_item_pill.vue

```diff
@@ -1,29 +1,29 @@
 <template>
-  <div :class="[wrapperClass, expanded ? 'dt-feed-item--expanded-pill' : 'd-bar-pill']">
-    <div :class="['d-bgc-secondary', expanded ? 'dt-feed-item--expanded-pill' : 'd-bar-pill']">
+  <div :class="[wrapperClass, pillClass]">
+    <div :class="['d-bgc-secondary', pillClass]">
       <div class="d-p8">
         <button
           :aria-label="ariaLabel"
           data-qa="dt-feed-item-pill"
-          :class="['d-baw0 d-bgc-moderate d-bar-pill d-w100p d-ta-left', buttonClass]"
+          :class="['d-baw0 d-bgc-moderate d-bar-pill d-w100p d-ta-left', toggleableClass, buttonClass]"
           @focusin="hover = true"
           @focusout="hover = false"
           @mouseenter="hover = true"
           @mouseleave="hover = false"
-          @click="$emit('click')"
+          @click="onClick"
         >
           <dt-item-layout class="d-w100p d-p8">
             {{ title }}
             <template slot="left">
               <dt-icon
                 v-if="toggleable && hover"
-                class="dt-feed-icon"
+                class="dt-feed-pill--icon"
```

Comment:
I think 

```suggestion
                class="dt-feed-item--pill-icon"
```

would be the right naming